### ISSUE TITLE
LabelSmoothingCrossEntropy

### ISSUE THREAD

**Issue by vainaijr** [pytorch/pytorch#24829]
## 🚀 Feature

label smoothing addition to standard cross entropy?

**Comment by SsnL** [comment]
duplicate of https://github.com/pytorch/pytorch/issues/11959
